q2_demux/_summarize/_visualizer.py

```diff
@@ -109,12 +108,15 @@ def summarize(output_dir: str, data: _PlotQualView, n: int=10000) -> None:
 
     per_sample_fastq_counts = {}
     reads = rev if not fwd and rev else fwd
+    id_reads = []
```

Comment:
I like that, it reflects some of the vocabulary we've used in other places in the code.  Thanks